### PR TITLE
feat: added pre-wired rule for IKS -> IS

### PR DESCRIPTION
**Pull request by Ak-sky**
### Description

Added pre-wired rule for IKS -> IS

### Release required?

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [x] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### Merge actions for mergers

- When merging, use a relevant [conventional commit](https://www.conventionalcommits.org/) message that is based on the PR contents and any release notes provided by the PR author. The commit message determines whether a new version of the module is needed, and if so, which semver increment to use (major, minor, or patch).
- Merge by using "Squash and merge".

### CODE REVIEW

File: examples/fscloud/variables.tf [hunk 1]
```diff
@@ -69,3 +69,9 @@ variable "allow_at_to_cos" {
   description = "Set rule for Activity Tracker to COS, default is true"
   default     = true
 }
+
+variable "allow_iks_to_is" {
+  type        = bool
+  description = "Set rule for IKS to IS, default is true"
```
Comment (vburckhardt) [inline examples/fscloud/variables.tf:75]: I think IS should be explain here as not fully obvious to someone not familiar with the underlaying CBR naming terminology.

File: modules/fscloud/README.md [hunk 2]
```diff
@@ -8,6 +8,7 @@ This module creates default coarse-grained CBR rules in a given account followin
 - VPCs where clusters are deployed -> COS
 - VPCs -> container registry
 - All ICD -> KMS
+- IKS -> IS
```
Comment (vburckhardt) [inline modules/fscloud/README.md:11]: Same comment as above on explaining IS

**Review by vburckhardt — CHANGES_REQUESTED** [review]

File: examples/fscloud/variables.tf (hunk 1 above)
Comment (Ak-sky) [reply examples/fscloud/variables.tf]: Done

File: modules/fscloud/README.md (hunk 2 above)
Comment (Ak-sky) [reply modules/fscloud/README.md]: Done

**Review by vburckhardt — APPROVED** [review]
lgtm

**Pull request merged**